api/config.py

```diff
@@ -37,6 +37,15 @@ class Configuration(CoreConfiguration):
     # of fines a patron can have before losing lending privileges.
     MAX_OUTSTANDING_FINES = u"max_outstanding_fines"
 
+    # The name of the per-library settings that set the maximum amounts
+    # of books a patron can have on loan or on hold at once.
+    # (Note: depending on distributor settings, a patron may be able
+    # to exceed the limits by checking out books directly from a distributor's
+    # app. They may also get a limit exceeded error before they reach these
+    # limits if a distributor has a smaller limit.)
```

Comment:
I think this explanatory text should go into the `description`s of these fields.